npm-shrinkwrap.json

```diff
@@ -2,11 +2,65 @@
   "name": "etcher",
   "version": "1.0.0-beta.18",
   "dependencies": {
+    "abbrev": {
```

Comment:
OK, looks not, but I have an alternative :) Can we register a global even handler for anchors and intercept that using `OSOpenExternalService`? This could be a `.run()` block in `lib/gui/os/open-external/open-external.js`.

For example:

```
$(document).on('click', 'a[href^="http"]', function(event) {
    event.preventDefault();
    OSOpenExternalService.open(this.href);
});
```

But we should find a non-jquery way of course.

By doing this it means that you no longer have to compile the HTML inside the modal, and you can use the built-in `ng-bind-html` and avoid the extra dependency altogether.